### PR TITLE
LPS-182231 Allow dragging items to the top of the tree

### PR DESCRIPTION
**Pull request by p2kmgcl**
The commits are already merged into master `LPS-181692 *`, the rest of them are some RTL fixes and some extra behavior improvements I found while fixing RTL.

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by p2kmgcl** [comment]
ci:test:sf

**Comment by p2kmgcl** [comment]
ci:test:relevant

File: modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/site_navigation_menu_editor/utils/useDragAndDrop.js [hunk 1]
```diff
@@ -205,6 +207,12 @@ function useNewDropTarget(item) {
 					return;
 				}
 
+				cardWidthRef.current =
+					cardWidthRef.current ||
+					targetRef.current
+						.querySelector('.card')
```
Comment (claraizquierdo) [inline modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/site_navigation_menu_editor/utils/useDragAndDrop.js:230]: I don't like very much to query by classes in the code, but maybe there is no other way.

**Review by claraizquierdo — COMMENTED** [review]
Hi @p2kmgcl,

everything seems to be working fine but, because of my lack of context I think it is better that other developer review it too.

File: modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/site_navigation_menu_editor/utils/useDragAndDrop.js (hunk 1 above)
Comment (p2kmgcl) [reply modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/site_navigation_menu_editor/utils/useDragAndDrop.js:230]: :thinking: you're right. Let me think if I can improve this a little bit :heart:

**Comment by victorg1991** [comment]
niquelao 👏

**Pull request closed without merging**